src/monitoring_service/database.py

```diff
@@ -13,11 +18,20 @@
     OnChainUpdateStatus,
 )
 
+SubEvent = Union[ActionMonitoringTriggeredEvent, ActionClaimRewardTriggeredEvent]
+
 log = structlog.get_logger(__name__)
 SCHEMA_FILENAME = os.path.join(
     os.path.dirname(os.path.realpath(__file__)),
     'schema.sql',
 )
+EVENT_ID_TYPE_MAP = {
```

Comment:
Maybe overkill for this simple case, but and `Enum` could be used here (along with `@unique`).

Reply:
I'd rather leave it for now. We can look into this again when we touch it.